src/test/scala/mockws/AuthenticationTest.scala

```diff
@@ -10,14 +10,14 @@ import Helpers._
 /**
  * Tests that [[MockWS]] simulates a WS client, in particular the methods involving authentication
  */
-class AuthenticationTest extends FunSuite with Matchers with PropertyChecks {
+class AuthenticationTest extends FunSuite with Matchers with PropertyChecks with BeforeAndAfterAll {
```

Comment:
I guess `with BeforeAndAfterAll` come from one experimentation?
If we do not use it, we can delete it now.